src/content/docs/en/guides/integrations-guide/db.mdx

```diff
@@ -129,26 +129,30 @@ There are a few shared configuration values across all columns:
 
 ### `indexes`
 
-Table indexes are used for improve lookup speeds on a given column or combination of columns. The `indexes` property accepts an object with a unique index name as the key:
+Table indexes are used for improve lookup speeds on a given column or combination of columns. The `indexes` property accepts an array of configuration objects specifying the columns to index:
 
-```ts title="db/config.ts" {6-8}
+```ts title="db/config.ts" {8-10}
```

Comment:
```suggestion
```ts title="db/config.ts" {9-11}
```
Not looking at a live preview, just counting on GitHub, but check this!

Reply:
Ah heck, added a column last minute and missed this. Thanks 🙈